src/EditorFeatures/TestUtilities/Workspaces/TestHostProject.cs

```diff
@@ -135,6 +136,11 @@ public string OutputFilePath
             get { return _outputFilePath; }
         }
 
+        public string DefaultNamespace
+        {
+            get { return _defaultNamespace; }
+        }
```

Comment:
nit: this could be an autoprop.  I get the tension here though given that the other fields/props are not.

Reply:
I will leave it as is just for consistency :)